homeassistant/components/accuweather/weather.py

```diff
@@ -73,6 +73,7 @@ def __init__(
             identifiers={(DOMAIN, coordinator.location_key)},
             manufacturer=MANUFACTURER,
             name=NAME,
+            configuration_url="https://www.accuweather.com",
```

Comment:
We should link to the specific location configured, as that information is available. (Deeplink when possible).

Reply:
Spent quite some times looking around the API, but unsure how I can get a link to the website - could do with a nudge

Reply:
Look at the website, open a location, see how the link is constructed. Use the location_key already available.

Reply:
Looks like you can just pass in `http://accuweather.com/en/_/_/{location_key}/weather-forecast/{location_key}/` and it'll auto-populate the URL